app/etl/item/clients/publishing_api.rb

```diff
@@ -0,0 +1,53 @@
+require 'gds_api/publishing_api_v2'
+
+class Item::Clients::PublishingAPI
```

Comment:
`PublishingAPI` is used in the `items` process. What are your thoughts about moving it there?

Reply:
Fixed in 6f512e2ec47733f83831527af491fcaac224590e